tests/utils/shared_tests.py

```diff
@@ -27,7 +28,10 @@ def attribute_quantity_valid(record, quantity):
     Domain Object attributes have been added since this test was written.
     Ensure tests have also been provided to validate all newly added
     attributes of this domain object."""
-    assert len(record) == quantity, error_message
+    pattern = "[a-z]+_field[0-9]+"
```

Comment:
It could be the case a non-dummy field is introduced which may match this pattern.